tests/test_utils_aggregation.py

```
@@ -0,0 +1,108 @@
+import unittest
```

There are two tests missing for the case that contradicting parameters are passed to the function:
* for metrics creation when passing a percentile and an aggregation function
* for groupings when passing several not None arguments besides `kind`

---

Both cases should raise an exception.